driver-sync/src/main/com/mongodb/client/Clock.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.client;
+
+/**
+ * A clock to wrap time operations.
+ *
+ * @since 3.11
+ */
+public interface Clock {
```

Comment:
Moved the interface to com.mongodb.client.internal.

Reply:
Looking again at this, we can remove the interface entirely, as no other class is accessing `ClientSessionClock` via the interface.  The interface would only be useful if, say, the `ClientSessionImpl` constructor were to take an instance of a `Clock`.  But since it's accessing the clock directly via `ClientSessionClock.INSTANCE`, there is no need of the abstraction.

Reply:
I have removed the Clock interface.